python/tk_multi_launchapp/prepare_apps.py

```diff
@@ -32,23 +33,24 @@ def prepare_launch_for_engine(engine_name, app_path, app_args, context, file_to_
     # Retrieve the TK Application instance from the current bundle
     tk_app = sgtk.platform.current_bundle()
 
-    try:
-        # Try to use the TK engine to perform the necessary preparations
+    # Make sure this version of core supports the create_engine_launcher method, this was introduced
+    # very recently, but we don't want to lock bugfixes to the legacy launch system behind a core upgrade.
+    if hasattr(sgtk.platform, "create_engine_launcher"):
```

Comment:
This `try/except` annoyingly swallowed an actual error in my code. AttributeError can be so common we should always ask for permission instead of forgiveness.